openapi/aro_hcp/v1alpha1/openapi.json

```diff
@@ -2058,6 +2058,13 @@
             "description": "The URL of the API server of the cluster.",
             "type": "string"
           },
+          "allowed_cidr_blocks": {
+            "description": "The CIDR blocks that are whitelisted for the API server. Currently, only supported for ARO-HCP.",
+            "type": "array",
+            "items": {
+              "type": "string"
```

Comment:
We should document the restrictions on the elements: length, format, ...

Reply:
Do we foresee information about the block to have more than just the IP as the information? I am wondering if the best would be an array of string or an array of struct if we foresee this having added information in the future